ui/libproject/devstart.rb

```diff
@@ -89,6 +96,12 @@ def clean_git_hooks()
 })
 
 Common.register_command({
+  :invocation => "rebuild-image",
```

Comment:
(Though my concern here is that somebody who modifies the docker file won't necessarily know they need to run a docker command in order to use it. A brief summary of when to use certain commands in the README wouldn't hurt.)